Handle Revolut CSV statement date format for English (UK) users

DragoonAethis opened the pull request:
English (UK) statements contain "Sept" instead of "Sep".

jczaja reviewed (commented):
Could you please add  a unit test for this fix . for example something like :   
https://github.com/RustInFinance/etradeTaxReturnHelper/blob/529f9fe58e0a36ce693847bef27bc4b7e0e14b58/src/csvparser.rs#L778

just "test_parse_transaction_dates_uk"  ? Alternatively you can strip your CSV file just to show example of transaction and send me over (jacek.czaja [at] gmail.com) so I can make UT myself

DragoonAethis commented:
OK, added a test with this case. It was the 6th copy of the same function, so I moved out the common code for these tests to a helper. It's probably not The Rust Way(TM) but seems a little better.

jczaja reviewed (commented):
Thanks for nice refactoring. All i good but CI related to code formatting failed. Please run "cargo fmt" and then commit changes it will make.

DragoonAethis commented:
Done

jczaja reviewed (approved):
LGTM

jczaja commented:
@DragoonAethis Many thanks for your contribution
